CONTRIBUTING.md

```diff
@@ -8,54 +8,63 @@ Please read through this document before submitting any issues or pull requests.
 
 ## Content
 
-- [❤️ Code of Conduct](#-code-of-conduct)
-- [Opening an issue](#opening-an-issue)
-- [How to work on an issue](#how-to-work-on-an-issue)
-- [How to make a pull request](#how-to-make-a-pull-request)
-  - [Developer Certificate of Origin](#developer-certificate-of-origin)
-  - [When is your pull request ready to be merged?](#-when-is-your-pull-request-ready-to-be-merged)
-- [How to merge a pull request](#how-to-merge-a-pull-request)
-- [Guideline commit messages](#guideline-commit-messages)
-  - [ℹ️ Semantic prefixes for commit messages](#-semantic-prefixes-for-commit-messages)
-  - [✍️ Writing a great commit message](#-writing-a-great-commit-message)
+- [Contributing Guidelines](#contributing-guidelines)
```

Comment:
I would remove this line and the [Content]
It's not valuable since it links to the file you're in or the section you're in. But adds additional and uncecessary depth to the table. That's why I started with the `Code of Conduct`

Reply:
That was autogenerated by the MD linter :/ Will remove it.